wormhole-connect/src/utils/errors.ts

```diff
@@ -41,11 +44,22 @@ export function interpretTransferError(
     } else if (USER_REJECTED_REGEX.test(e?.message)) {
       uiErrorMessage = 'Transfer rejected in wallet, please try again';
       internalErrorCode = ERR_USER_REJECTED;
-      /* TODO SDKV2
-    } else if (e.message === SWAP_ERROR) {
-      uiErrorMessage = SWAP_ERROR;
-      internalErrorCode = ERR_SWAP_FAILED;
-      */
+    } else if (
+      transferDetails.route.includes('CCTP') &&
+      /burn.*exceed/i.test(e?.toString())
+    ) {
+      // As of this code being written the CCTP limit is 1,000,000 USDC in a single transfer
+      // It's possible Circle could change this in the future and we're not reading the limit
+      // from their contracts dynamically for now so we assume it's 1M and tell users that if
+      // their amount exceeded 1M
+      const assumedCircleLimit = 1_000_000;
+      const { amount } = transferDetails;
+      const limitString =
+        amount !== undefined && amount > assumedCircleLimit
```

Comment:
Nitpick: You probably don't need to check ` amount !== undefined` if you are checking `amount > assumedCircleLimit`

Reply:
yes I do https://github.com/wormholelabs-xyz/wormhole-connect/blob/9ce47c0d8010800afa765e1dc2266d85252261c0/wormhole-connect/src/telemetry/types.ts#L18

Reply:
@artursapek what I meant was if the amount is undefined `amount > assumedCircleLimit` will always be falsy :)

Reply:
Not a blocker 👍

Reply:
I understood what you meant. like I said TypeScript is complaining if I remove this check:

<img width="561" alt="image" src="https://github.com/user-attachments/assets/677e1510-a97f-4456-bd91-7cb4a0f20b70">